### PR TITLE
docs: Add documentation and example for QwikIntrinsicElements

### PR DESCRIPTION
**Pull request by hbendev**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [x] Docs / tests / types / typos

# Description

Adds missing documentation and JSDoc comment for the `QwikIntrinsicElements` interface. Related issue: #4148.

# Checklist:

- [x] My code follows the [developer guidelines of this project](https://github.com/BuilderIO/qwik/blob/main/CONTRIBUTING.md)
- [x] I have performed a self-review of my own code
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by shairez** [comment]
Thanks @hbendev !

Please run 
```
pnpm api.update
pnpm fmt.staged
```
and commit the fixes
thanks!

File: packages/docs/src/routes/api/qwik/index.md [hunk 1]
```diff
@@ -1214,6 +1214,24 @@ export interface QwikFocusEvent<T = Element> extends SyntheticEvent<T, NativeFoc
...
+The interface holds available attributes of both native DOM elements and custom Qwik elements. An example showing how to define a customizable wrapper component:
+
+```tsx
+import { component$, Slot, type QwikIntrinsicElements } from "@builder.io/qwik";
+
+type WrapperProps = {
+  attributes?: QwikIntrinsicElements["div"];
+};
+
+const Wrapper = component$<WrapperProps>(({ attributes }) => {
```
Comment (zanettin) [inline packages/docs/src/routes/api/qwik/index.md:1226]: just a thought - maybe we could use an  `export default component$(...)` here instead to make it a working example?

**Review by zanettin — COMMENTED** [review]
nice job and thanks a lot 🙏  welcome in the contributors familiy 🎉

File: packages/docs/src/routes/api/qwik/index.md (hunk 1 above)
Comment (hbendev) [reply packages/docs/src/routes/api/qwik/index.md:1226]: Sure, I can make that change soon.

**Comment by zanettin** [comment]
thanks a lot 🙏

**Pull request merged**